Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -218,57 +220,34 @@ void WriteSwitch(ShaderCode& out, APIType ApiType, std::string_view variable,
                  const Common::EnumMap<std::string_view, last_member>& values, int indent,
                  bool break_)
 {
-  const bool make_switch = (ApiType == APIType::D3D);
-
   // The second template argument is needed to avoid compile errors from ambiguity with multiple
   // enums with the same number of members in GCC prior to 8.  See https://godbolt.org/z/xcKaW1seW
   // and https://godbolt.org/z/hz7Yqq1P5
   using enum_type = decltype(last_member);
 
-  // {:{}} is used to indent by formatting an empty string with a variable width
-  if (make_switch)
```

Comment:
Do we know why switch statements were avoided on OGL/Vk?  We have switch statements elsewhere, and I had no trouble with glslang's handling of switch statements here when working on the Metal renderer, and I think they look much nicer than the alternative massive if/else mess.